docs/sources/reference/api/docker_remote_api.md

```diff
@@ -46,6 +46,22 @@ You can still call an old version of the API using
 
 ### What's new
 
+`GET /containers/(id)/copy`
+
+`PUT /containers/(id)/copy`
```

Comment:
> `PUT` also gives wrong impression as this endpoint isn't idempotent.

You're right, I guess it's not idempotent in certain cases (like when it has to create a directory when copying from another)